x-pack/plugins/fleet/public/hooks/use_request/outputs.ts

```diff
@@ -17,6 +19,21 @@ export function useGetOutputs() {
   });
 }
 
+export function useDefaultOutput() {
+  const outputsRequest = useGetOutputs();
+  const output = useMemo(() => {
+    return outputsRequest.data?.items.filter((o) => o.is_default)[0];
```

Comment:
minor: is there always at most one default output? code could be shortened items.find(o => o.is_default)

Reply:
Yes you are right it will be a lot more elegant (and perfomant too) with a `.find`